.devcontainer/devcontainer.json

```
@@ -0,0 +1,25 @@
+{
+	"name": "Hello",
```

```suggestion
	"name": "imap_processing",
```
or something more descriptive?